src/components/board/PostingDetail.js

```diff
@@ -78,19 +94,49 @@ const PostingDetail = () => {
 export default PostingDetail;
 
 const Container = styled.div`
-  max-width: 700px;
-  margin: auto 50px;
+  width: 80%;
+  margin: 10px auto;
+  height: 15rem;
+`;
+
+const Form = styled.form`
+  margin: 10px auto;
+  margin-bottom: 50px;
```

Comment:
```suggestion
  margin: 10px auto 50px auto;
```

이렇게 한 줄로 작성할 수도 있을 것 같습니다..!